### DISCUSSION TITLE
UU hottop 2k+ and Mac book M1

### DISCUSSION THREAD

**Discussion by HarrisBurdick** [artisan-roaster-scope/artisan#725] [Q&A] [answered]
I am having trouble connecting my machine. I followed through the install instructions and start up as best as possible but I am still having no reading ( uu ) for both temps. I have been working through the instructions over and over but am not having any change in results. I have read a few other people have had similar problems but they were able to switch computers and I don't have that option. Any help is great.

**Comment by MAKOMO** [comment, marked as answer]
Did you install the Phidget "Standalone Control Panel" on the top of this page: "https://www.phidgets.com/docs/OS_-_macOS" or the regular driver a bit further down under "OS X 10.11+: Installer Download"?

**Reply by HarrisBurdick** [reply]
Thank you very much.
